site/assets/scss/_component-examples.scss

```diff
@@ -228,6 +228,12 @@
   }
 }
 
+.bd-example-rounded-utils {
```

Comment:
Maybe change this to the generic `.bd-example` class and use a child selector?

```scss
.bd-example > [class^="rounded"] { ... }
```

Would avoid a new selector and parent class.

Reply:
Doesn't it feel stricter with the extra parent class? TBH I think it's better this way...

Reply:
The suggestion I made matches most of the `.bd-example` customizations in this file. I'd like to take a crack at updating it after this because most of it is triggering off the parent class, which is fine to me and avoids writing extra HTML just to have our docs.